src/left-nav.jsx

```diff
@@ -181,7 +181,25 @@ const LeftNav = React.createClass({
           onTouchTap={this._onOverlayTouchTap} />
       );
     }
-
+    let children;
+    if (this.props.menuItems === undefined) {
+      children = this.props.children;
+    }
+    else {
+       children = <Menu
```

Comment:
Can you follow https://github.com/airbnb/javascript/tree/master/react#parentheses?